plugins/ruby-foreman-openscap/debian/control

```diff
@@ -0,0 +1,13 @@
+Source: ruby-foreman-openscap
+Section: ruby
+Priority: extra
+Maintainer: Foreman Openscap Team
+Build-Depends: debhelper, foreman (>= 1.12.0~rc1), ruby-foreman-deface (<<2.0.0)
```

Comment:
there were a lot of changes in 0.6, I'd prefer nightly for a while and then we can move to stable branch